gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -471,6 +483,41 @@ gst_tensor_trainer_change_state (GstElement * element,
   return GST_STATE_CHANGE_FAILURE;
 }
 
+
+/**
+ * @brief Check if current epochs is complete, 
+ * tensor_trainer wait for one of epochs to complete before getting the results from the subplugin
+ */
+static gboolean
+gst_tensor_trainer_epochs_is_complete (GstTensorTrainer * trainer)
```

Comment:
this is ```wait-for-completion```, not ```is-complete```.

Reply:
okay, thank you.

Reply:
Is @myungjoo's comment resolved?

Reply:
yes, I kept the function name and created the wait function below.